templates/resources/securityhub_finding_aggregator.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Aggregate Findings From All Regions
+
```

Comment:
- Could you include a note in text here that the list of regions cannot contain the calling region in the list ?
- I do believe there is an issue on the finding_aggregator if we switch the `region_linking_mode` from one listing the regions to one without.

Reply:
@quixoticmonk Thanks. I will do some testing this week and update the examples as appropriate.